xbmc/platform/posix/main.cpp

```diff
@@ -41,6 +44,42 @@
 #include "input/linux/LIRC.h"
 #endif
 #include "platform/XbmcContext.h"
+  
+#if defined(TARGET_POSIX)
```

Comment:
yes, please. never trust existing code :)
with move to cmake most of those guards became obsolete